pytorch_lightning/loggers/__init__.py

```diff
@@ -24,40 +24,25 @@
     'CSVLogger',
 ]
 
-try:
-    # needed to prevent ImportError and duplicated logs.
-    environ["COMET_DISABLE_AUTO_LOGGING"] = "1"
+from pytorch_lightning.loggers.comet import _COMET_AVAILABLE, CometLogger
+from pytorch_lightning.loggers.mlflow import _MLFLOW_AVAILABLE, MLFlowLogger
+from pytorch_lightning.loggers.neptune import _NEPTUNE_AVAILABLE, NeptuneLogger
+from pytorch_lightning.loggers.test_tube import _TESTTUBE_AVAILABLE, TestTubeLogger
+from pytorch_lightning.loggers.wandb import _WANDB_AVAILABLE, WandbLogger
 
-    from pytorch_lightning.loggers.comet import CometLogger
-except ImportError:  # pragma: no-cover
-    del environ["COMET_DISABLE_AUTO_LOGGING"]  # pragma: no-cover
-else:
+if _COMET_AVAILABLE:
     __all__.append('CometLogger')
+    # needed to prevent ImportError and duplicated logs.
+    environ["COMET_DISABLE_AUTO_LOGGING"] = "1"
```

Comment:
```suggestion
if _COMET_AVAILABLE:
    __all__.append('CometLogger')
    # needed to prevent ImportError and duplicated logs.
    environ["COMET_DISABLE_AUTO_LOGGING"] = "1"
else:
    del environ["COMET_DISABLE_AUTO_LOGGING"]  # pragma: no-cover 
   
```

Reply:
why do you want to delete something which is not set in the first place?

Reply:
It could be set while running the script `COMET_DISABLE_AUTO_LOGGING=1 python train.py`. It was in the code before . This PR removes it. If you don't think it is needed, then ignore this comment.

Reply:
moreover, most likely it would crash as the key is missing in the env...